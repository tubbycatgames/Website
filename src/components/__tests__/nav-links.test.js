import React from "react"
import renderer from "react-test-renderer"

import NavLinks from "../nav-links"

describe("Nav Links", () => {
  it("renders app nav links", () => {
    const tree = renderer.create(<NavLinks />)
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
