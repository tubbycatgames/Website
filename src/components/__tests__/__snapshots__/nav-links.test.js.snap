// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Links renders app nav links 1`] = `
<div
  className="links"
>
  <a
    href="/"
  >
    <p>
      About
    </p>
  </a>
  <a
    href="/games"
  >
    <p>
      Games
    </p>
  </a>
</div>
`;
